cmd/protoc-gen-go-grpc/main.go

```diff
@@ -58,6 +61,7 @@ func main() {
 	var flags flag.FlagSet
 	requireUnimplemented = flags.Bool("require_unimplemented_servers", true, "set to false to match legacy behavior")
 	useGenericStreams = flags.Bool("use_generic_streams_experimental", true, "set to true to use generic types for streaming client and server objects; this flag is EXPERIMENTAL and may be changed or removed in a future release")
+	useSeparateServicePackage = flags.Bool("use_separate_service_package", false, "set to true to use separate service package, service package will import protobuf package")
```

Comment:
How about something like: name this `output_package` and make it a string?  Then the import path could be `file.GoImportPath` with everything after the last `/` stripped and replaced by what's specified here?  That would allow more flexibility for how the code is generated.

Reply:
The import path is useless after file generate. 

https://github.com/grpc/grpc-go/pull/8245.
https://github.com/grpc/grpc-go/issues/8233#issuecomment-2816990330

Reply:
I think the most user friendly solution to split package would be to have a proto option like  `go_grpc_package` which has semantics similar to `go_package`. The probuf complier would then need to handle splitting the service and message code when `go_package` and `go_grpc_package` are different.

Reply:
Is that something we can do?  I thought the options were defined by protobuf.  We could propose a `go_services_package` if so.  The Go protobuf team seems to prefer this ideology anyway (generate services and messages into separate packages).

Reply:
This would be my preference, then.

We should think about whether renaming the package completely would be better than just adding a suffix.  I'm not sure which one would be better, but either would be better than the previous proposal.